app/knowledge_panels.py

```diff
@@ -1,167 +1,191 @@
+        if self.value is not None:
+            path += f"/{self.value}"
+            description += f" {self.value}"
+        # Checking if secondary facet is provided
+        if self.sec_facet is not None:
+            path += f"/{self.sec_facet}"
+            description += f" {self.sec_facet}"
+        if self.sec_value is not None:
+            path += f"/{self.sec_value}"
+            description += f" {self.sec_value}"
+        (quality_html, source_url, t_description, t_title) = data_quality(url=url, path=path)
 
-    return {
-        "Quality": {
-            "title": t_title,
-            "subtitle": f"{t_description} {description}",
-            "source_url": f"{source_url}/data-quality",
-            "elements": [
-                {
-                    "element_type": "text",
-                    "text_element": quality_html,
-                }
-            ],
-        },
-    }
+        return {
+            "Quality": {
+                "title": t_title,
+                "subtitle": f"{t_description} {description}",
+                "source_url": f"{source_url}/data-quality",
+                "elements": [
+                    {
+                        "element_type": "text",
+                        "text_element": quality_html,
+                    }
+                ],
+            },
+        }
 
+    def last_edits_kp(self):
+        """
+        Return knowledge panel for last-edits corresponding to different facet
+        """
+        query = {
+            "fields": "product_name,code,last_editor,last_edit_dates_tags",
+            "sort_by": "last_modified_t",
+        }
+        description = ""
+        if self.facet == "country":
+            self.country = self.value
+            country_code = country_to_ISO_code(value=self.value)
+            url = f"https://{country_code}-en.openfoodfacts.org"
+            self.facet = self.value = None
+        if self.country is not None:
+            country_code = country_to_ISO_code(value=self.country)
+            url = f"https://{country_code}-en.openfoodfacts.org"
+            description += f"{self.country} "
+        if self.country is None:
+            url = "https://world.openfoodfacts.org"
+        if self.facet is not None:
+            description += f"{self.facet}"
+        if self.value is not None:
+            query[f"{facet_plural(facet=self.facet)}_tags_en"] = self.value
+            description += f" {self.value}"
+            source_url = f"{url}/{self.facet}/{self.value}?sort_by=last_modified_t"
+        if self.sec_value and self.sec_facet is not None:
+            query[f"{facet_plural(facet=self.sec_facet)}_tags_en"] = self.sec_value
+            description += f" {self.sec_facet} {self.sec_value}"
+            source_url = f"{url}/{self.facet}/{self.value}/{self.sec_facet}/{self.sec_value}?sort_by=last_modified_t"  # noqa: E501
+        expected_html, t_description, t_title = last_edit(url=url, query=query)
```

Comment:
minor:
expected_html is a strange name (it's for test that we use this expression).
It's more, edits_html or t_html.